lib/webauthn/credential_options.rb

```diff
@@ -9,5 +9,9 @@ class CredentialOptions
     def challenge
       @challenge ||= SecureRandom.random_bytes(CHALLENGE_LENGTH)
     end
+
+    def timeout
+      @timeout = WebAuthn.configuration.credential_options_timeout
```

Comment:
I guess the idea was this to be

```ruby
@timeout ||= WebAuthn.configuration.credential_options_timeout
```
?

Anyway, I think i'll drop using memoization here.
No need to cache the value I think.